pages/reference/OS/advanced.md

```diff
@@ -5,94 +5,91 @@ excerpts: Configuration options to expose more device functionality to {{ $names
+__Note:__ After modifying a `config.txt` variable, the device supervisor will apply the changes and reboot the device.
 
-For example, setting the value of `RESIN_HOST_CONFIG_gpu_mem` to 16 will produce the following entry in `config.txt`:
+Variables that start with the `BALENA_HOST_CONFIG_` or `RESIN_HOST_CONFIG_` prefix will be added to the `config.txt` file, replacing any preexisting values in the file. For example, a variable named `BALENA_HOST_CONFIG_start_x` with the value of `1` will result in the following entry in `config.txt`:
 
 ```
-gpu_mem=16
+start_x=1
 ```
 
-These variables can be set using the API or any of its clients, including the [SDK][sdk], the [CLI][cli], and the dashboard. To configure via the dashboard, add variables to the *Fleet Configuration* page for fleet-wide config variables and to *Device Configuration* for device specific variables. The device-specific variables, if defined, will override the fleet-wide variables of the same name.
+To manage configuration variables via the dashboard, add/edit variables via the *Fleet Configuration* page for [fleet-wide config variables][config-vars-fleet] and *Device Configuration* for [device specific variables][config-vars-device]. The device-specific variables, if defined, will override the fleet-wide variables of the same name. See the [configuration variables documentation][config-vars] for more details.
+
+Configuration values that are prepopulated may be edited or toggled (enabled/disabled):
+
+<img alt="Prepopulated device configuration variables" src="/img/configuration/default_configuration.png" width="100%">
 
-**After modifying a config.txt variable, the device supervisor will apply the changes and reboot the device.**
+Custom configuration variables may be added by clicking _Add custom variable_ and entering a name with the prefix of `BALENA_HOST_CONFIG_` or `RESIN_HOST_CONFIG_` as shown below:
 
-__Note:__ Configuration variables defined through the API will not apply to devices in [local mode][local-mode].
+<img alt="Add fleet configuration variable" src="/img/configuration/variable_editor_config.png" width="60%">
+
+__Note:__ In addition to the dashboard, these configuration variables can be also be set using the API or any of its clients, including the [SDK][sdk] and [CLI][cli].
 
 ### GPU Memory
 
-You can configure the amount of memory shared with the GPU by adding entries to
-`config.txt` as follows:
+The [amount of memory][gpu-memory] that is addressable from the GPU my be configured by adding entries to `config.txt`. You can target different memory sizes, specified in MB, of 256, 512, and 1024 corresponding to the RAM on the Raspberry Pi. Values will be ignored for those devices whose memory size does not match. For the Raspberry Pi 4, which has versions with RAM greater than 1GB, the minimum and maximum values are the same as for a 1GB device.
```

Comment:
"You can target different memory sizes, specified in MB, of 256, 512, and 1024 corresponding to the RAM on the Raspberry Pi." doesn't really make much sense - perhaps it could be reworded? Maybe something like "You can also set specific values of gpu_mem for Raspberry Pis with 256, 512 or 1024 MB of RAM." ?